src/Sylius/Bundle/AdminBundle/Twig/ChannelNameExtension.php

```diff
@@ -36,6 +39,8 @@ public function getFilters(): array
 
     public function getChannelNameByCode(string $code): string
     {
+        @trigger_error('Function getChannelNameByCode is deprecated since Sylius 1.7 and will be removed in Sylius 2.0.', \E_USER_DEPRECATED);
```

Comment:
If it will be merged to 1.6 then: 
```suggestion
        @trigger_error('Function getChannelNameByCode is deprecated since Sylius 1.6 and will be removed in Sylius 2.0.', \E_USER_DEPRECATED);
```